csharp_webapi/2.EchoBot-With-Counter/EchoBot.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+
+namespace EchoBotWithCounter
+{
+    /// <summary>
+    /// Represents a bot that processes incoming activities.
+    /// For each user interaction, an instance of this class is created and the OnTurnAsync method is called.
+    /// This is a Transient lifetime service.  Transient lifetime services are created
```

Comment:
>This is a Transient lifetime service [](start = 8, length = 36)

I suspect the discussions around "Transient lifetime service" are *only* valid for ASP.Net Core 2. Consider replacing with the WebAPI relevant text around lifecycle management.